src/garage/tf/algos/dqn.py

```diff
@@ -191,32 +191,28 @@ def train_once(self, itr, paths):
 
         """
         paths = self.process_samples(itr, paths)
-        epoch = itr / self.steps_per_epoch
+        epoch = itr // self.steps_per_epoch
 
         self.episode_rewards.extend(paths['undiscounted_returns'])
         last_average_return = np.mean(self.episode_rewards)
         for _ in range(self.n_train_steps):
             if (self.replay_buffer.n_transitions_stored >=
                     self.min_buffer_size):
-                self.evaluate = True
                 qf_loss = self.optimize_policy(epoch, None)
                 self.episode_qf_losses.append(qf_loss)
 
-        if self.evaluate:
-            if itr % self.target_network_update_freq == 0:
-                self._qf_update_ops()
+        if itr % self.target_network_update_freq == 0:
+            self._qf_update_ops()
```

Comment:
A flag is needed so qf is not updated during the first `min_buffer_size` steps. Previously it's guarded by `self.evaluate`.